scripts/linters/python_linter.py

```diff
@@ -367,8 +408,10 @@ def _lint_py_files(self, config_pylint, config_pycodestyle):
 
             if pylinter.msg_status != 0 or pycodestyle_report.get_count() != 0:
                 summary_message = stdout.getvalue()
+                pylint_messages = (
+                    self._get_trimmed_summary_message(pylint_output.read()))
```

Comment:
Ditto is this only for error message? if yes please highlight this in the function name!

Reply:
Done.